install.sh

```diff
@@ -168,16 +166,8 @@ function setup_script () {
 
 # Install basic apt and flatpak packages
 function so_packages () {
-  sudo apt update -y
-  sudo apt upgrade -y
```

Comment:
Removing `upgrade` command to reduce execution time. 

Leaving upgrades for the user to deal with.